src/plugins/preauth/pkinit/pkinit_crypto_openssl.c

```diff
@@ -4957,7 +4949,7 @@ crypto_retrieve_X509_key_usage(krb5_context context,
     if (ret_eku_bits)
         *ret_eku_bits = 0;
     else {
-        pkiDebug("%s: EKUs not requested, not checking\n", __FUNCTION__);
+        TRACE_PKINIT_NO_EKU_CHECK(context);
```

Comment:
Never happens (only one call site which never passes NULL).  Same for the next message.  The conditionals should be removed instead, although that's not necessarily in scope for this PR.